config/routes.rb

```diff
@@ -3,7 +3,9 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
```

Comment:
please discard your changes here to avoid conflict after merging you can do it again